index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="src/styles.css" />
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet"
+        integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link href="https://fonts.googleapis.com/css2?family=Merriweather+Sans:ital,wght@0,300;0,400;1,400&display=swap"
+        rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/1113260fee.js" crossorigin="anonymous"></script>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
+
+    <title>weather</title>
+</head>
+
+<body>
+    <div class="app">
+        <div class="container">
+            <div class="container">
+                <div class="row">
+                    <div class="col-7">
+                        <span id="res">14</span> <a href="#" id="cel">°C</a>|<a href="#" id="fahr">°F</a>
+                        <br />
+                        Humidity: <span id="hum">59</span>%
+                        <br />
+                        Wind: <span id="wind">6 </span> m/sec
+                        <br />
+                        <span id="description">Clouds</span>
+
+                        <img src="" alt="Clear" id="icon" class="float-left" />
+                    </div>
+
+                    <div class="col-4">
+                        <span id="city"> Kyiv </span>
+                        <br />
+                        <div id="currDate">6/23/2022</div>
```

Comment:
as you saw in bootstrap naming for id / class, they do someting like element-feature convention, like curr-date.
Also please don't do abbreviated names, so do current-date instead of curr-date